City-of-Helsinki/drupal-hdbt
# UHF-2085 Errand service and service channel layout

How to test:

1. Make sure you have imported services, errand services and service channels from TPR.
2.  If you have them already just skip to part 9. If not run `make shell`
3. `drush en -y helfi_tpr_config`
4. Import some TPR entities (set `MIGRATE_LIMIT=500 `or similar if seen fit):
5. `drush migrate:import tpr_service`
6. `drush migrate:import tpr_errand_service`
7. `drush migrate:import tpr_service_channel`
8. `drush cr` Now you should have some services and errand services and service channels.
9. Checkout out required branches for your local: `composer require drupal/helfi_tpr:dev-UHF-2085_errand_service_and_service_channel_layout && composer require drupal/helfi_hdbt:dev-UHF-2085_errand_service_and_service_channel_layout`
10. Go to view a service - for example: https://helfi.docker.so/fi/asukaspysakointi It should now have errand services and service channels styled according to these layouts: https://share.goabstract.com/d59ede6a-c821-4f86-ac28-e75a3810a74b?collectionLayerId=21c38c53-d3a5-423c-a322-71d853b88de9&mode=design&sha=latest

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/46
